README.md

```diff
@@ -97,6 +97,9 @@ $ make build
 $ make run-local
 ```
 
+## Packit architecture
```

Comment:
I don't think this is interesting to people that much, please start an architecture doc in docs/ and put it there